code/game/turfs/space/space.dm

```diff
@@ -15,7 +15,8 @@
 	var/global/datum/gas_mixture/space/space_gas = new
 	plane = PLANE_SPACE
 
-/turf/open/space/New()
+/turf/open/space/Initialize()
+	..()
```

Comment:
One thing you can do is not call parent from here.

That should lead to a drop in ss init time.

(The parent call has always been intentionally left out from here.)